src/dpr/components/filter-controls/view.njk

```diff
@@ -0,0 +1,43 @@
+{% from "govuk/components/details/macro.njk" import govukDetails %}
```

Comment:
Is this a bit bound to this first step - the filter section? When you/we come to do the Show Columns work, is this going to be re-used or duplicated?

Personally, I think it might be good to look at it as an accordion component, with the content provided by the page. Or possible an accordion component with a parent component for the whole filter/columns section.

What do you think?